dtfs-central-api/src/types/db-models/banks.d.ts

```diff
@@ -1,5 +1,10 @@
 import { WithId } from 'mongodb';
 
+type ReportPeriodSchedule = {
```

Comment:
I think this should be `UtilisationReportPeriodSchedule`

Reply:
I thought about it but if we stored any other reporting schedules they could also be the same type